plugins/capture_space/services/CaptureSpaceService.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @service captureSpace
+ * @package plugins.captureSpace
+ * @subpackage api.services
+ */
+class CaptureSpaceService extends KalturaBaseService
+{
+	/**
+	 * Returns latest version and URL
+	 *
+	 * @action clientUpdates
+	 * @param string $os
+	 * @param string $version
+	 * @return KalturaCaptureSpaceUpdateResponse
+	 * @throws CaptureSpaceErrors::ALREADY_LATEST_VERSION
+	 * @throws CaptureSpaceErrors::NO_UPDATE_IS_AVAILABLE
+	 */
+	function clientUpdatesAction ($os, $version)
+	{
+		if(kCaptureSpaceVersionManager::isLatest($os, $version)){
```

Comment:
I can't find a file with name such as (k)CaptureSpaceVersionManager  in this branch,
am I missing something?